auto_dev/commands/run.py

```diff
@@ -15,33 +15,49 @@
 import rich_click as click
 from docker.errors import NotFound
 from aea.skills.base import PublicId
+from aea.configurations.base import PackageType
 
 from auto_dev.base import build_cli
-from auto_dev.utils import map_os_to_env_vars
+from auto_dev.utils import map_os_to_env_vars, load_autonolas_yaml
 from auto_dev.constants import DOCKERCOMPOSE_TEMPLATE_FOLDER
 from auto_dev.cli_executor import CommandExecutor
 
 
 cli = build_cli()
 
 
+class SupportedLedger(Enum):
```

Comment:
i think this might even be a core enum.
We have an enums file, 
Can we move this there.